core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -368,8 +356,7 @@ private[spark] object JsonProtocol {
     ("Shuffle Read Metrics" -> shuffleReadMetrics) ~
     ("Shuffle Write Metrics" -> shuffleWriteMetrics) ~
     ("Input Metrics" -> inputMetrics) ~
-    ("Output Metrics" -> outputMetrics) ~
-    ("Updated Blocks" -> updatedBlocks)
```

Comment:
I looked at this some more and I don't see any issue with this.  As you said below on 842 its an option and I tested this both ways with history server as well just to be sure.  new history file (without "Updated blocks" entry) with old history server and old file (with "Updated blocks" entry in history file) with new history server, both work fine.

if you think we should leave it I can put it back with a empty value?